### DISCUSSION TITLE
Remove foreign project from repositories list

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22342] [New to GitHub]
<p>Hi,</p>
<p>I worked with VuePress (<a href="https://github.com/vuejs/vuepress" rel="nofollow noopener">https://github.com/vuejs/vuepress</a>) and opened an issue which got already closed. The repository itself still appears in my repositories list.</p>
<p><img src="https://user-images.githubusercontent.com/10137/181070260-74b16747-f137-49a1-8b0e-2847d5e04b1e.png" alt="unknown" data-base62-sha1="he3QYHxdGs7rkgRyYHI0VIyzElW" width="333" height="133"></p>
<p>I would like to remove it from this list. I headed up to <strong><a href="https://github.com/settings/repositories" rel="nofollow noopener">https://github.com/settings/repositories</a></strong> and tried to remove it manually but it did not appear in this list.</p>
<p>Is it possible to remove it or will it always appear because I started an issue some time ago?</p>

**Comment by mpboom** [comment]
<p>Are you on the new beta version of the dashboard?</p>

**Comment by ghost** [comment]
<p><a class="mention" href="/mpboom">@mpboom</a> yes I am</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/garzec">@garzec</a>,</p>
<p>Thanks for being here. I spent a bit of time talking with my coworkers about this, and I have some information that I hope will help.</p>
<p>First of all, that repository list sorts based on most recent activity. So, if you had recently contributed to that repo, it would be showing up there. That’s expected behavior.</p>
<p>The list does update automatically as you make new contributions to other repos. So, as long as you don’t make anymore contributions to the vuepress repo, you should see its name fall further down the “Respositories” list over time.</p>
<p>As for manually removing it, that isn’t currently possible. The dashboard view where you see that “Repositories” list is only viewable to you, and again, the list is sorted automatically based on recent activity. That all said, I can understand why you may like to remove a recently contributed to repository from that list, and I will share your feedback with the team for future consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by ghost** [comment]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> well I didn’t know that but I think this “feature” is really nice :slight_smile: Thanks a lot for the explanation. I’m fine with that :slight_smile:</p>
